client/gutenberg/extensions/navigation/editor.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Wordpress dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+import { registerBlockType } from '@wordpress/blocks';
+import { InnerBlocks } from '@wordpress/editor';
+
+/**
+ * External dependencies
+ */
+
+import classnames from 'classnames';
+
+ /**
+ * Internal dependencies
+ */
+
+import './style.scss';
+import './editor.scss';
+import Navigation from './navigation.js';
+import config from './config.js';
+
+const { name, title, icon, category, keywords, attributes, baseClasses } = config;
+registerBlockType( name, {
+	title: __( title ),
+	icon: icon,
+	category: category,
+	keywords: keywords.map( keyword => __( keyword ) ),
```

Comment:
sadly this translation won't work because the underlying system relies on extracting string literals from the source-code.